src/main/java/util/JsonHelper.java

```diff
@@ -0,0 +1,71 @@
+package util;
+
+import com.google.gson.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+/**
+ * A helper that manages conversion from objects to JSON Strings, and vice versa.
+ */
+public class JsonHelper {
+    private static final Logger logger = LogManager.getLogger(JsonHelper.class.getName());
+
+    private Gson gson;
+
+    /**
+     * Sets up the JsonHelper by initialising the Gson component
+     */
+    public JsonHelper() {
+        setupGson();
+    }
+
+    /**
+     * Configures the gson object that will serialise and deserialise json data
+     */
+    private void setupGson() {
+        gson = new GsonBuilder()
+                .setPrettyPrinting()
+                .registerTypeAdapter(LocalDateTime.class,
+                        (JsonSerializer<LocalDateTime>) (src, typeOfSrc, context) -> {
+                            Instant instant = src.atZone(ZoneId.systemDefault()).toInstant();
+                            long epochMilli = instant.toEpochMilli();
+                            return new JsonPrimitive(epochMilli);
+                        })
+                .registerTypeAdapter(LocalDateTime.class,
+                        (JsonDeserializer<LocalDateTime>) (json, typeOfT, context) -> {
+                            Instant instant = Instant.ofEpochMilli(json.getAsJsonPrimitive().getAsLong());
+                            return LocalDateTime.ofInstant(instant, ZoneId.systemDefault());
+                        })
+                .create();
+    }
+
+    /**
+     * Creates an instance of T from a JSON-formatted String
+     * @param json The String in JSON format that contains the values of an instance of T
```

Comment:
More like `JSON in String format`? Can be more concise, `JSON representation of instance of T in String format`.